README.md

```diff
@@ -1 +1,83 @@
-Placeholder for Checkstyle Regression Tool readme.
+# Regression-tool
+
+Regression-tool is an automation tool to do regression testing based on proposed patch(Pull Request) 
+of checkstyle project.
+
+## Setup
+
+### Clone
+
+Just clone this repository to your local.
+
+### Generate Jar
+
+Running the following command, then a `regression-tool-XX-all.jar` would be generated in the `target` folder.
+
+```bash
+$ mvn clean package -Passembly
+```
+
+## Usage
+
+```bash
+$ java -jar regression-tool-XX-all.jar -p <arg> -r <arg> [--stopAfterGenerateConfig] [-t <arg>]
+```
+
+### Arguments
+
+#### localGitRepo(r)
+
+The path of the checkstyle repository.
+
+- Required
+- One arguement

```

Comment:
These `-` doesn't really look good on the actual page view, imo.
https://github.com/Luolc/regression-tool/blob/60715d51a016ce2aa04b3279a41873650c88d4c8/README.md#localgitrepor

Why don't we just make them sentences on the end of the main description.